### DISCUSSION TITLE
Add Substack icon as part of links sections

### DISCUSSION THREAD

**Discussion by robertosilviu** [jpanther/congo#514] [Ideas]
It could be useful to have the support for substack icon inside the section **links** of `languages.en.toml` file. If i add a substack link to my profile it shows an invisible icon. Thank you for this awesome theme.

**Comment by leonidasv** [comment]
You can add your own custom icons to your `assets/icons` folder: https://jpanther.github.io/congo/samples/icons/

After that, you can use the icon filename in the `languages.en.toml` just as you do for built-in icons.

Tip: if you want your icon to work seamlessly for both dark and white colorschemes, add `fill="currentColor"` to SVG's `path`. This is not documented AFAIK, I've discovered by myself. Maybe it should be added to the documentation...
